.env.example

```
@@ -14,4 +14,8 @@ SERVER_PORT=8080
 SWAGGER_ENABLED=true
 
 CORS_ALLOWED_ORIGINS=http://localhost:3000
-CORS_ALLOWED_METHODS=GET,PUT,POST,PATCH,DELETE,OPTIONS
\ No newline at end of file
+CORS_ALLOWED_METHODS=GET,PUT,POST,PATCH,DELETE,OPTIONS
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CORS_ALLOWED_METHODS key should go before the CORS_ALLOWED_ORIGINS key

(UnorderedKey)